config/admin-functions/lockdown.sh

```diff
@@ -2,6 +2,14 @@
 set -euo pipefail 
 
 : "${VX_FUNCTIONS_ROOT:="$(dirname "$0")"}"
+: "${VX_CONFIG_ROOT:="/vx/config"}"
+
+# detect Surface Go
+if [ $(cat ${VX_CONFIG_ROOT}/machine-type) == "bsd" ]; then
+    surface=1
+else
+    surface=0
+fi
```

Comment:
this approach is a little bit brittle as it assumes that `bsd` == Surface Go.

Can we instead do the hardware detection like we do in other scripts to determine if the hardware is a Surface Go, independently of what software it's running?

Reply:
Yes, I had it configured this way so I could test in a VM. Will fix.